client/reader/following/intro.jsx

```diff
@@ -35,34 +36,48 @@ class FollowingIntro extends React.Component {
 
 	render() {
 		const { isNewReader, translate, dismiss, isNewUser } = this.props;
-		const linkElement = <a onClick={ this.props.handleManageLinkClick } href="/following/manage" />;
 
 		if ( ! isNewReader || ! isNewUser ) {
 			return null;
 		}
 
+		const linkElement = <a onClick={ this.props.handleManageLinkClick } href="/following/manage" />;
+
+		// A/B test three variants of the new illustration
+		const abtestVariant = abtest( 'readerIntroIllustration' );
```

Comment:
the `abtest` function is weirdly heavy. might want to make this a module level variable instead of rerunning it on every `render` call